_config/site.yml

```diff
@@ -40,7 +40,7 @@ profile_base: &profile
     :minify: *minify
     :cdn_version: *cdn_version
   authenticate_google_books_api: true
-  build_threads: 40
+  build_threads: 0
```

Comment:
Agreed! 

I was just doing it temporarily to get a faraday log that wasn't interlaced. 


> On 5 Mar 2015, at 21:27, Jason Porter <notifications@github.com> wrote:
> 
> In _config/site.yml:
> 
> > @@ -40,7 +40,7 @@ profile_base: &profile
> >      :minify: *minify
> >      :cdn_version: *cdn_version
> >    authenticate_google_books_api: true
> > -  build_threads: 40
> > +  build_threads: 0
> This has to stay, otherwise our build times go through the roof.
> 
> —
> Reply to this email directly or view it on GitHub.
>